File: Java-Programming-Practise/Java-tutorial-in-hindi/While_Loop.java
package com.java;

import java.util.Scanner;

public class While_Loop {
    public static void main(String[] args) {
        int input, i=1;

        Scanner scanner = new Scanner(System.in);

        System.out.println("How many time you want the loop to run? ");

        input = scanner.nextInt();

        while (input >= i){
            System.out.println(i);
            i++;
        }
    }
}

File: C-Programming-Practise/Programming-Tutorial-For-Beginners-Learn-C/while_loop.c
#include <stdio.h>
int main()
{
    int num=1, input;
    scanf("%d", &input);
    printf("how many times you want the loop to run?\n");

    while(num<=input)
    {
        printf("%d\n", num);
        num++;
    }
    
    return 0;
}